WeakAurasTemplates/TriggerTemplates.lua

```diff
@@ -824,63 +883,28 @@ local function subTypesFor(item, regionType)
                   isOnCdGrey(conditions, 1, regionType);
                   overlayGlow(conditions, 2, regionType);
                 end,
+                data = dataGlow
               });
             end
           end
         end
-        if (item.requiresTarget) then
-          tinsert(types, {
-            icon = icon.target,
-            title = L["Show Cooldown and Check for Target"],
-            description = L["Always shows the aura, turns red when out of range."],
-            createTriggers = function(triggers, item)
-              createAbilityTrigger(triggers, 1, item, "showAlways");
-            end,
-            createConditions = function(conditions, item, regionType)
-              insufficientResourcesBlue(conditions, 1, regionType);
-              isOnCdGrey(conditions, 1, regionType);
-              isSpellNotInRangeRed(conditions, 1, regionType);
-            end,
-          });
-          if (item.overlayGlow) then
-            tinsert(types,  {
-              icon = icon.glow,
-              title = L["Show Cooldown and Check for Target & Proc Tracking"],
-              description = L["Always shows the aura, highlight while proc is active, turns red when out of range, blue on insufficient resources."],
-              createTriggers = createAbilityAndOverlayGlowTrigger,
-              createConditions = function(conditions, item, regionType)
-                insufficientResourcesBlue(conditions, 1, regionType);
-                isOnCdGrey(conditions, 1, regionType);
-                isSpellNotInRangeRed(conditions, 1, regionType);
-                overlayGlow(conditions, 2, regionType);
-              end,
-            });
-          end
-        elseif (item.overlayGlow) then
-          tinsert(types,  {
-            icon = icon.glow,
-            title = L["Show Cooldown and Proc Tracking"],
-            description = L["Always shows the aura, highlight while proc is active, blue on insufficient resources."],
-            createTriggers = createAbilityAndOverlayGlowTrigger,
-            createConditions = function(conditions, item, regionType)
-              insufficientResourcesBlue(conditions, 1, regionType);
-              isOnCdGrey(conditions, 1, regionType);
-              overlayGlow(conditions, 2, regionType);
-            end,
-          });
-        end
       end
     end
   elseif(item.type == "buff") then
+    data.inverse = false
+    dataGlow.inverse = false
     tinsert(types, {
       icon = icon.cd,
       title = L["Show Only if Buffed"],
       description = L["Only shows the aura if the target has the buff."],
       createTriggers = function(triggers, item)
         createBuffTrigger(triggers, 1, item, "showOnActive", true);
       end,
-      data = { inverse = false },
+      data = data,
     });
+    if subglowindex then
+      data.subRegions = { [subglowindex] = subglow }
```

Comment:
I don't understand this. Isn't the only difference between data and dataGlow that dataGlow has the glow sub region?
So essentially, aren't after this data and dataGlow identically?